books/std/util/defaggregate.lisp

```diff
@@ -1117,6 +1125,7 @@ would have had to call @('(student->fullname x)').  For instance:</p>
     :parents
     :short
     :long
+    :suppress-xdoc
```

Comment:
Suggest the polarity of the option be reversed, to avoid double negation.  Something like :generate-xdoc with default t.

Reply:
The problem with reversing is that it generates automatically.  I think suppressing is the right concept.